Cargo.toml

```
@@ -78,7 +78,9 @@ rusoto_dynamodb = { version = "0.47", default-features = false, features = [
 sentry = { version = "0.31", features = [
   "debug-logs",
 ] } # Using debug-logs avoids https://github.com/getsentry/sentry-rust/issues/237
-sentry-core = { version = "0.31" }
+sentry-core = { version = "0.31", default-features = false, features = [
+  "client",
+] }
```

I think this was in another PR as well -- we don't use this feature AFAICT

```suggestion
sentry-core = { version = "0.31" }
```